### PR TITLE
Party Chat + Add BedDestroyer credit for final kill

### PR DESCRIPTION
**Pull request by ClydeSan-25933**
- Added Party Chat (Need Improvement)
- Added Kill credit goes to Bed Destroyer if No killer/last hit player found

### CODE REVIEW

File: bedwars-plugin/src/main/java/com/tomkeuper/bedwars/listeners/chat/ChatFormatting.java [hunk 1]
```diff
@@ -54,6 +55,15 @@ public void onChat(AsyncPlayerChatEvent e) {
         if (e == null) return;
         Player p = e.getPlayer();
 
+        if (PartyCommand.chatToggle.getOrDefault(p.getUniqueId(), false) && BedWars.getPartyManager().hasParty(p)) {
+            // Send the message only to party members
+            e.getRecipients().clear();
+            e.getRecipients().addAll(BedWars.getPartyManager().getMembers(p));
+            String format = ChatColor.translateAlternateColorCodes('&', "&e&lPARTY &8&l┃ &b%bw_playername% &7→ ") + ChatColor.GRAY + "%bw_message%";
```
Comment (tomkeuper) [inline bedwars-plugin/src/main/java/com/tomkeuper/bedwars/listeners/chat/ChatFormatting.java:62]: Do we want the format to be configurable with the language file?

**Comment by ritana5** [comment]
I think this should be added.

**Comment by tomkeuper** [comment]
> I think this should be added.

agreed but its not finished....

**Comment by Azerith-MC** [comment]
Hmm if want i can complete it and send

**Comment by ClydeSan-25933** [comment]
> > I think this should be added.
> 
> agreed but its not finished....

If you want i can complete it and push a new commit (might take some time)

**Comment by ritana5** [comment]
I think you should

**Comment by tomkeuper** [comment]
how much is some time? Otherwise I will do it myself soon(tm) :D

**Comment by ClydeSan-25933** [comment]
> how much is some time? Otherwise I will do it myself soon(tm) :D

Sorry about that i really can't work on it anytime soon due to my own issues irl. Most of the things shall work with some fixes and messages in lang files.

**Comment by tomkeuper** [comment]
> > how much is some time? Otherwise I will do it myself soon(tm) :D
> 
> Sorry about that i really can't work on it anytime soon due to my own issues irl. Most of the things shall work with some fixes and messages in lang files.

All good, Ill implement it as is. And fix issues on the go! In case you wanto to add more just add them later :D

**Review by tomkeuper — APPROVED** [review]

**Pull request merged**